src/dataflow/src/render/debezium.rs

```diff
@@ -59,6 +59,14 @@ pub(crate) fn render<G: Scope>(
                                     }
                                     None => continue,
                                 };
+
+                                // Ignore out of order updates that have been already overwritten
```

Comment:
Would the test fail without this? Or: why didn't we do this before?

Reply:
It started failing in the iteration of this PR that shuffled the data. I could remove it now since the decode exchange is making sure to exchange by key but since it's more Correct (tm) I think i'll leave it there